### ISSUE TITLE
15. API - SPI Devices states is misleading

### ISSUE THREAD

**Issue by Milliways2** [gpiozero/gpiozero#1080]
15. API - SPI Devices states

"The gpiozero library provides two SPI implementations:

…

A hardware based implementation. This is only available when the SPI kernel module is loaded, and the Python spidev library is available. It can only use specific pins for SPI communication (GPIO11=clock, GPIO10=MOSI, GPIO9=MISO, while GPIO8 is select for device 0 and GPIO7 is select for device 1). However, it is extremely fast and works with all devices."

SPI1, with 3 hardware chip selects, is available on all Raspberry Pis with 40 pin header
 
SPI2, also with 3 hardware chip selects, is only available on Compute Module 1, 3 and 3+

On the Raspberry Pi 4, 400 and Compute Module 4 there are 4 additional SPI buses:
       SPI3 to SPI6, each with 2 hardware chip selects

**Comment by lurch** [comment]
Yes, see e.g. #1030 and #1016  (i.e. this hasn't been updated / tested for the additional SPI interfaces on the Pi 4)
